api/src/server.ts

```diff
@@ -13,16 +17,29 @@ import {
   PORT
 } from './utils/env';
 
-const requestSerializer = (request: FastifyRequest) => ({
-  method: request.method,
-  url: request.url,
-  ip: request.headers['x-forwarded-for'] || request.ip,
-  hostname: request.hostname,
-  remoteAddress: Array.isArray(request.headers['x-forwarded-for'])
-    ? request.headers['x-forwarded-for'][0]
-    : request.headers['x-forwarded-for'] || request.ip,
-  remotePort: request.socket.remotePort
-});
+const requestSerializer = (req: FastifyRequest) => {
+  const method = req.method || 'METHOD not found';
+  const url = req.url || 'URL not found';
+  const headers = req.headers || 'HEADERS not found';
+  const xForwardedFor = Array.isArray(req.headers['x-forwarded-for'])
+    ? req.headers['x-forwarded-for'][0]
+    : req.headers['x-forwarded-for'];
+  const ip =
+    xForwardedFor || req.headers['x-real-ip'] || req.ip || 'IP not found';
+  const query = isEmpty(req.query) ? 'QUERY not found' : req.query;
+  const hostname = req.hostname || 'HOSTNAME not found';
+  const remotePort = req.socket.remotePort || 'REMOTE_PORT not found';
+
+  return {
```

Comment:
```suggestion
  return {
    REQ_ID: req.id,
```

Include the request ID